### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #8 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#349]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 4.91 KB...]
 T E S T S
-------------------------------------------------------

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 17, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
#
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00007f2cf6b2f000, 262144, 0) failed; error='Cannot allocate memory' (errno=12)
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 262144 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/hs_err_pid14951.log

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:11 min
[INFO] Finished at: 2017-05-17T09:12:11+00:00
[INFO] Final Memory: 33M/206M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/jacoco.exec -jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefirebooter8093696394100136114.jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefire1981394632526039217tmp /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefire_02053533866479772181tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Arthur Sapozhnikov] bfe27ab27f315d90a622d209ee0436e1c8b70dfd - Added reviews to the modules

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/8/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
